src/app/chat/view.nim

```diff
@@ -253,8 +253,6 @@ QtObject:
 
     self.activeChannel.setChatItem(selectedChannel)
     self.status.chat.setActiveChannel(selectedChannel.id)
-    discard self.status.chat.markAllChannelMessagesRead(selectedChannel.id)
-    self.currentSuggestions.setNewData(self.status.contacts.getContacts())
     self.setLastMessageTimestamp(true)
     self.activeChannelChanged()
```

Comment:
You could even remove those 2 lines as well since `setActiveChannel` performs them as well

Reply:
Oh I didn't even notice these. Just focused on RPC. Fixed!